examples/simpleapp/src/main/resources/sc_settings.conf

```diff
@@ -22,11 +22,15 @@ scorex {
         agentName = "2-Hop"
     }
 
-    websocket {
+    websocketClient {
+        enabled = true
```

Comment:
Here we may setup it to `false` because it is the config file with already predefined genesi info and no expected MC connection.

Reply:
![image](https://user-images.githubusercontent.com/110401079/185977813-27bbb83b-6da3-4210-890e-89351ded76fb.png)